tests/unit/test_algebraic_expression.cpp

```diff
@@ -341,6 +341,177 @@ TEST_F(AlgebraicExpressionTest, AlgebraicExpression_New) {
     AlgebraicExpression_Free(operation);
 }
 
+TEST_F(AlgebraicExpressionTest, AlgebraicExpression_Domains) {	
+	GrB_Matrix A;
+	GrB_Matrix B;
+    const char *src;
+    const char *dest;
```

Comment:
Add comments about operations domains:
`The addition operation is done over matrices with the same source and destination domain`
`Multiplication source domain is the first operand row domain, and destination domain is the last operand column domain.`
`The domain names A and B represent the tuple <row domain, col domain> of the matrices A and B`
also, rename src and dest to srcDomain, destDomain

Reply:
👍